gnc/sub8_perception/nodes/marker_occ_grid.py

```diff
@@ -10,7 +10,6 @@
 from image_geometry import PinholeCameraModel
```

Comment:
ok, doing

On Sat, May 7, 2016 at 10:25 AM, mattlangford <notifications@github.com>
wrote:

> In gnc/sub8_perception/nodes/marker_occ_grid.py
> <https://github.com/uf-mil/Sub8/pull/126#discussion_r62416966>:
>
> > @@ -10,7 +10,6 @@
> >  from image_geometry import PinholeCameraModel
>
> If you wanted to move this file as part of this PR, go ahead
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/uf-mil/Sub8/pull/126/files/70803231d6847ed5bff27658560f6b4a10c0749e#r62416966>
>